src/hooks/useBlock.js

```diff
@@ -4,14 +4,16 @@ import useInterval from './useInterval'
 import { useWallet } from '../providers/Wallet'
 
 export function useLatestBlock(updateEvery = 1000) {
-  const { getBlockNumber } = useWallet()
+  const { ethers } = useWallet()
```

Comment:
i think we just need the `getBlock`

Reply:
The `getBlock` function is part of  `ethers`.  `useWallet` provides the `getBlockNumber` function but couldn't make it work that's why i changed it